net6-react/ClientApp/src/components/case/Employees.jsx

```diff
@@ -32,34 +34,184 @@ import { Todo } from "./Todo";
+    } else {
+      setSelectedBoss();
+    }
+  };
+
   return (
-    <div
-      style={{
-        display: "flex",
-        flexDirection: "column",
-        gap: 16,
-      }}
-    >
-      <h1>Brukere i organisasjonen</h1>
+    <div>
+      {loading ? (
+        <div>
+          <p>Loading</p>
+        </div>
+      ) : (
+        <div>
+          <h1>Brukere i organisasjonen</h1>
+          {/* {empData.map((employee) => {
+            return (
+              <div key={employee.id} onClick={() => handleClick(employee)}>
+                <Employee Employee={employee} />
+              </div>
+            );
+          })} */}
 
-      <Todo what="Fjerne denne" how="Fjern denne todoen">
-        <strong>For resten av todoene er det bare å putte løsningen inni komponenten.</strong>
-        <div>Slik som dette</div>
-      </Todo>
+          {/* <Todo what="Fjerne denne" how="Fjern denne todoen">
+            <strong>
+              For resten av todoene er det bare å putte løsningen inni
+              komponenten.
+            </strong>
+            <div>Slik som dette</div>
+          </Todo> */}
 
-      <Todo what="Vise sjefen" how="Vis sjefen for organisasjonen her med <Employee />-komponenten" />
+          <Todo
+            what="Vise sjefen"
+            how="Vis sjefen for organisasjonen her med <Employee />-komponenten">
+            {CEO && (
+              <div
+                style={{ border: "2px solid black" }}
+                onClick={() => handleClick(CEO)}>
+                <Employee Employee={CEO} />
+              </div>
+            )}
+          </Todo>
 
-      <Todo what="Vise hvem folk rapporterer til" how="Folk som har en sjef, gruppert på sjef. Vis navn på sjefen, og <Employee /> for den ansatte " />
+          <Todo
+            what="Vise hvem folk rapporterer til"
+            how="Folk som har en sjef, gruppert på sjef. Vis navn på sjefen, og <Employee /> for den ansatte ">
+            {leader && (
+              <div>
+                {Object.keys(leader).map((key) => {
+                  return (
+                    <div key={key}>
+                      <h1>{employee[key].name}</h1>
+                      {leader[key].map((e) => {
+                        return (
+                          <div
+                            style={{ border: "2px solid black" }}
+                            key={e.id}
+                            onClick={() => handleClick(e)}>
+                            <Employee Employee={e} />
+                          </div>
+                        );
+                      })}
+                    </div>
+                  );
+                })}
+              </div>
+            )}
+          </Todo>
 
-      <Todo what="Gutta på gulvet" how="<Employee /> for alle ansatte der sjefen har en sjef" />
+          <Todo
+            what="Gutta på gulvet"
+            how="<Employee /> for alle ansatte der sjefen har en sjef">
+            {bossHasBoss && (

```

Comment:
Unødvendig conditional, teknisk umulig for bossHasBoss å være falsey.
Dette gjelder for alle memos/states som ikke initialiseres som null på noe tidspunkt.

Reply:
Har gjort "bossHasBoss" på nytt. Logikken jeg originalt hadde var feil, men den skal være rett nå.